examples/autoignition-mpi.py

```diff
@@ -135,10 +137,10 @@ def main(ctx_factory=cl.create_some_context, use_logmgr=True,
     constant_cfl = False
 
     # i.o frequencies
-    nstatus = 1
-    nviz = 5
-    nhealth = 1
-    nrestart = 5
+    nstatus = 100
+    nviz = 100
+    nhealth = 100
+    nrestart = 100
```

Comment:
Very minor comment --- should we at least ensure these callbacks are called at least once when the example run in CI? That way we can avoid any potential surprises down the line if a callback in an example driver has a potential issue

Reply:
Yes, this squish probably snuck in on accident.  I'll fix it up.